gogetaPollers.go

```diff
@@ -1,51 +1,50 @@
 package main
```

Comment:
General note: Since we're moving to a single process some of these files will be messy (with comments). Wanted to have a logical break in our source between refactoring the queues and implementing a one shot system.

Also I've given into golang and started using tabs on gofmt - so will be converting this whole project over to the whims of Google.